.github/workflows/update-bff-dependency/src/file-helper.js

```diff
@@ -0,0 +1,69 @@
+const core = require("@actions/core");
+
+class FileHelper {
+  #repositoryHelper;
+
+  constructor(repositoryHelper) {
+    this.#repositoryHelper = repositoryHelper;
+  }
+
+  async updateOpenApiToolsFile(branchName, directory, tags) {
+    let file;
+    try {
+      file = await this.#repositoryHelper.getFileContent(
+        branchName,
+        `${directory}/openapitools.json`
+      );
+    } catch (error) {
+      core.info("No OpenAPI file to update");
+      return [];
+    }
+
+    try {
+      let parsedContent = JSON.parse(file);
+      let fileUpdated = false;
+
+      const generators = parsedContent["generator-cli"].generators;
+      for (const [generatorName, generator] of Object.entries(generators)) {
+        const inputSpecUrl = generator.inputSpec;
+        const urlParts = inputSpecUrl.split("/");
```

Comment:
instead of split, we can use a regular expression? Check how it was done in the bff action